content/features/featuresDeepDive/animation/animation_method.md

```diff
@@ -37,26 +37,54 @@ const myAnim = new BABYLON.Animation(name, property, frames_per_second, property
     BABYLON.Animation.ANIMATIONLOOPMODE_YOYO - The animation will reverses its direction when it reaches the end instead of restarting from the beginning
 
 ## Set Key Frames
-This is an array, *myKeys* of objects. Each object has the following two properties:
+The key frames of an animation are held in an array of objects, each containing the following two properties:
 
 - _frame_ - the frame number
 - _value_ - for the property being changed
 
-Once constructed this is added to the animation.
+This array will contain one object for each key frame and can contain as many key frames as needed for the desired animation.
+
+```javascript
+const myKeys = [
+    {
+        frame: 0,
+        value: 0.5
+    },
+    {
+        frame: 60,
+        value: 1.0
+    }
+];
+```
+
+Once the key frames array is constructed, in this case an array named *myKeys*, it is added to the animation by setting the keys.
 
 ```javascript
 myAnim.setKeys(myKeys);
 ```
 
 ## Beginning The Animation
 
-To run the animation it is pushed onto the *animations* array property of the mesh
+There are several ways to manage the play state of animations. One of the simplest is to add an animation to the *animations* array property of an object. Objects like Mesh, Material, Light, or Camera have an *animations* array property defined by default which holds animations which target that specific object.  
 
 ```javascript
-mesh.animations.push(myAnim)
+myMaterial.animations.push(myAnim)
 ```
 
-and started with these required parameters
+Additionally, animations can be added to any object that the engine can reach, even if it does not already have an *animations* array property. Due to the malleable nature of Javascript, simply declaring a new array property for an object will enable us to store animations on target objects. For example, if we want to target the weight property of an animation with another animation, we can add an *animations* array property to the animation by declaring it.
+
+```javascript
+myAnimation.animations = [];
+myAnimation.animations.push(weightAnimation);
+
+```
+
+Remember, above we said that anything that the engine can reach can be animated with our animation system. This also applies to content outside of the canvas. For example, since we can reach any DOM object with Javascript, we can apply an animation to it. If we wanted to, we could use an animation to drive the border-radius of a `<div>` object.
+
+### beginAnimation
+
+Now that we have pushed an animation to a target object, starting the animation is a simple case of telling the scene to begin the animations stored with the target object.

```

Comment:
Maybe we should add a sentence to make clear that `beginAnimation` will actually start all animations stored in the `target.animations` array. It may not be obvious from the name of the method...

Reply:
updated in https://github.com/BabylonJS/Documentation/pull/985/commits/a1bb49597e5396b9df7d5758de73276390ccbd5e